File-Formats/PDF/Convert-HTML-To-PDF/blazor.md

```diff
@@ -11,16 +11,16 @@ keywords: Assemblies
 
 Steps to convert HTML to PDF in Blazor application
 
-Create a new C# Blazor application project. Select Blazor App from the template and click the Next button.
+1. Create a new C# Blazor application project. Select Blazor App from the template and click the Next button.
 ![Blazor_step1](htmlconversion_images/blazor_step1.png)
 
 Now, the project configuration window appears. Click Create button to create a new project with the default project configuration.
```

Comment:
In the project configuration window, name your project and select _Create_.

Reply:
modified the line